nixos/modules/config/console.nix

```diff
@@ -168,6 +168,7 @@ in
           "${config.boot.initrd.systemd.package.kbd}/bin/setfont"
           "${config.boot.initrd.systemd.package.kbd}/bin/loadkeys"
           "${config.boot.initrd.systemd.package.kbd.gzip}/bin/gzip" # Fonts and keyboard layouts are compressed
+          "${config.boot.initrd.systemd.package.kbd.gzip}/bin/.gzip-wrapped"
```

Comment:
Would it be valuable to add this location to the comment introduced in https://github.com/NixOS/nixpkgs/pull/211948?

https://github.com/NixOS/nixpkgs/blob/d29b1ecb89246c592ce53d8a8326358edec23ab0/pkgs/tools/compression/gzip/default.nix#L46

Reply:
@cole-h While that makes sense in general, I sort of question the validity of that comment in the first place. Are we really going to keep a list of all the places that refer to `.gzip-wrapped` in that comment? Why that one but not any other wrapped executables? e.g. We have an explicit reference to `.systemd-cryptsetup-wrapped` for the same reason, but no comment.

What are your thoughts?

Reply:
Fair enough.

I was thinking that this would be a short-lived workaround, so knowing what and where to remove would be valuable, but it might end up like `.systemd-cryptsetup-wrapped` and stick around.

Basically, I don't care / don't think it matters either way!